src/data/ua_overrides.js

```diff
@@ -726,29 +726,31 @@ const AVAILABLE_UA_OVERRIDES = [
     /*
      * Bug 1743429 - Add UA override for sites broken with the Version 100 User Agent
      *
-     * We're running an experiment on Desktop with Beta and Nightly, to investigate
-     * how much the web breaks with a Version 100 User Agent. Some sites do not
-     * like this, so let's override for now
+     * Some sites have issues with a UA string with Firefox version 100 or higher,
+     * so present as version 99 for now.
      */
     id: "bug1743429",
```

Comment:
This is technically the wrong metabug, as it was for the v100 experiment. You can ignore this, I will file a new metabug in my cycle and adjust this accordingly - I'm just leaving a comment so I have a link to add to my todo list... :)